agda2hs.agda-lib

```diff
@@ -1,4 +1,4 @@
 name: agda2hs
 depend:
 include: lib
-flags: --erase-record-parameters
```

Comment:
Why is this not needed anymore? I thought we want erased parameters to be actually erased.

Sure, for Sets/Levels/Sizes we have no annotations anymore, but what about everything else?

Or is it simply that just our Prim/Prelude modules don't require them, and it's up to the user to put this flag in case they use erased record parameters?

Reply:
Yeah exactly, the primitive modules no longer require the flag, and the flag actually caused problems when I was trying to use Agda2Hs together with --erased-cubical.